### ISSUE TITLE
The agent doesn't handle IgnoreAttribute very well that is assigned to TestFixture

### ISSUE THREAD

**Issue by aliaksandrbasau** [reportportal/agent-net-nunit#9]
To reproduce:
1. Add the following test fixture class with Ignore attribute:
```
  [TestFixture]
  [Ignore("Ignore until V2 is sufficiently deployed")]
  [Category("Functional")]
  public class BackendHumanResourcesTests : BaseFixture
  {
  ....
```
2. Configure RP integration and run the test fixture. => There will a test item on RP that is not properly finished.
![image](https://cloud.githubusercontent.com/assets/6681200/26008112/6904c7da-3711-11e7-8a95-8d6df9a59f94.png)
![image](https://cloud.githubusercontent.com/assets/6681200/26008118/73dfc7b8-3711-11e7-9e41-0ab51fcad574.png)

**Comment by nvborisenko** [comment]
@aliaksandrbasau Alex, this is default behavior of Report Portal for skipped tests. By default skipped test is marked as to investigate.

Do you see any reasons to override this behavior by agent?

**Comment by aliaksandrbasau** [comment]
@nvborisenko I would override it in this case since there is no reason to investigate the ignored tests as I made a conscious decision to ignore them.
I don't know about pure NUnit integration but ignored tests are not shown on RP in case of SpecFlow+NUnit.

**Comment by nvborisenko** [comment]
I saw 2 issues here:
1) Ignored TestFixture should not be "To Investigate" (agent should mark this fixture as skipped with a reason)
2) Launch is still "in progress" state.

In any case this should be resolved on agent side.

Alex, please prove my understanding.

**Comment by aliaksandrbasau** [comment]
@nvborisenko You are correct in both cases.
And making ignored fixtures as Skipped should be fine (or even preferable over not displaying the ignored fixtures on RP at all).

**Comment by nvborisenko** [comment]
Agent works good here, launch finishes correctly.
The issue is related to incorrect representation of empty suites on RP side.
![screenshot_1](https://user-images.githubusercontent.com/22616990/31944143-c0c9e254-b8d3-11e7-95cd-b7c10566da98.png)
@DzmitryHumianiuk again, RP doesn't handle empty suites.

**Comment by DzmitryHumianiuk** [comment]
@nvborisenko @aliaksandrbasau can you even know, that it will be empty?

we will introduce NestedStep class, which will not be considered for statistics.

cc @avarabyeu

**Comment by aliaksandrbasau** [comment]
@DzmitryHumianiuk As I understand it's not known that the test item will be empty when we add it. The exception will be thrown by nunit later.

**Comment by DzmitryHumianiuk** [comment]
@aliaksandrbasau can you handle this exception and delete empty suite?

**Comment by nvborisenko** [comment]
Looks like a workaround. Applying it on agent side is not very good solution.

**Comment by heaventud** [comment]
Nikolay, as I understand, the problem with ignored testfixtures still exsits ?

**Comment by nvborisenko** [comment]
Yes, RP cannot handle empty suites very well. @DzmitryHumianiuk what is our plan in resolving this issue? Example of incorrect launch is [here](https://rp.epam.com/ui/#default_project/userdebug/all%7Cpage.page=1&page.size=50&page.sort=start_time,number%2CDESC/5b34ceffb9d82100013344cb?page.page=1&page.size=50&page.sort=start_time%2CASC)

**Comment by DzmitryHumianiuk** [comment]
@nvborisenko in 5.0

**Comment by DzmitryHumianiuk** [comment]
Fully closed with #16 ?

**Comment by nvborisenko** [comment]
Not fully :) Really empty suite is not handled by RP. This issue can be considered as fixed.
